### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by boegaderrick**
Added new storage engine methods (get & count) and their unit tests.

### CODE REVIEW

File: tests/test_models/test_engine/test_db_storage.py
Comment (Itsfoss0) [inline tests/test_models/test_engine/test_db_storage.py:1]: You've imported the `pycodestyle` package but not using it anywhere in the unittest

**Review by Itsfoss0 — APPROVED** [review]
Looking great @boegaderrick. Nice test cases by the way. 
Ready to be merged 🥳

File: tests/test_models/test_engine/test_db_storage.py
Comment (Itsfoss0) [reply tests/test_models/test_engine/test_db_storage.py:1]: You could find a minute or two to look into that. Otherwise, everything else is great.

Comment (boegaderrick) [reply tests/test_models/test_engine/test_db_storage.py:1]: Thank you for pointing that out, I'll delete that line.

**Pull request merged**